tests/DateTest.php

```diff
@@ -15,4 +15,22 @@ protected function getTestSubjectType()
     {
         return 'date';
     }
+
+    public function testDateTimeValuesAreBoundAsFormattedStrings()
+    {
+        $dateTimeLocal = new Date('dob');
+        $dateTimeLocal->defaultValue(new DateTime('12-04-1988 10:33'));
+
+        $expected = '<input type="date" name="dob" value="1988-04-12">';
+        $this->assertSame($expected, $dateTimeLocal->render());
+    }
+
+    public function testDateTimeDefaultValuesAreBoundAsFormattedStrings()
+    {
+        $dateTimeLocal = new Date('dob');
+        $dateTimeLocal->defaultValue(new DateTime('12-04-1988 10:33'));
```

Comment:
I went for date as that resembles the classname even more. Hope that's fine.
Thank you!